Misc/NEWS.d/next/Library/2025-06-02-22-23-38.gh-issue-135056.yz3dSs.rst

```diff
@@ -0,0 +1,2 @@
+Add a ``--header`` cli option to :program:`python -m http.server`. Contributed by
```

Comment:
```suggestion
Add a ``--header`` CLI option to :program:`python -m http.server`. Contributed by
```

Reply:
Fixed in 5a30d914ad194bfac400bbaa5e699bd77f9785ed